recipes/mgs-canopy/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "mgs-canopy" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_rev: 00a60e0
+  git_url: https://github.com/fplaza/mgs-canopy-algorithm.git
```

Comment:
Unfortunately, there is no proper tarball (`<name>-<version>.tar.gz`) available in the Github repo